raspbian/updating.md

```diff
@@ -36,7 +38,7 @@ The kernel and firmware are installed as a Debian package, and so will also get
 
 ### Running out of space
 
-When running `sudo apt-get dist-upgrade`, it will show how much data will be downloaded and how much space it will take up on the SD card. It's worth checking with `df -h` that you have enough free disk space, as unfortunately `apt` will not do this for you. Also be aware that downloaded package files (`.deb` files) are kept in `/var/cache/apt/archives`. You can remove these in order to free up space with `sudo apt-get clean`.
+When running `sudo apt dist-upgrade`, it will show how much data will be downloaded and how much space it will take up on the SD card. It's worth checking with `df -h` that you have enough free disk space, as unfortunately `apt` will not do this for you. Also be aware that downloaded package files (`.deb` files) are kept in `/var/cache/apt/archives`. You can remove these in order to free up space with `sudo apt clean`.
```

Comment:
One difference in behaviour between `apt` and `apt-get` is that `apt` will remove the downloaded archives afterwards. So the last two sentences are not necessary.